### PR TITLE
Rating component

### PR DESCRIPTION
**Pull request by i-am-a-codegirl**
Please review my rating component

this page contains the rating component of CSS

live link: https://css-companion.netlify.app/components/rating.html

Following are the features of this page

- nav: contains brand name and links to the home page and components library.
- main: contains the documentation of rating components with snippets and examples.
- aside: contains links to all my component libraries.
- footer: contains links to my social media
image

### CODE REVIEW

File: component/rating/rating.css [hunk 1]
```diff
@@ -0,0 +1,572 @@
...
+}
+
+.container{
+    color: var(--primary-color);
+    width: 100vw;
+    text-decoration: none;
+    display: grid;
+    font-weight: 400;
+    grid-auto-flow: column;
+    grid-template-columns: 200px 1fr;
```
Comment (sahiba0915) [inline component/rating/rating.css:25]: avoid using absolute units

File: component/rating/rating.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./rating.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+
+    <script src="https://kit.fontawesome.com/e45a62e93a.js" crossorigin="anonymous"></script>
+</head>
+    <title>Document</title>
```
Comment (sahiba0915) [inline component/rating/rating.html]: give a proper title

**Review by sahiba0915 — COMMENTED** [review]
work on suggestions rest looks good to go

File: component/rating/rating.css [hunk 3]
```diff
@@ -0,0 +1,572 @@
...
+
+    background: #111827;
+}
+
+aside label{
+    color: var(--black);
+}
+
+aside #hamburger{
+grid-area: ham;
```
Comment (pnchinmay) [inline component/rating/rating.css:119]: take care of indentation
better use beautify or any other css linters in editor

File: component/rating/rating.css [hunk 4]
```diff
@@ -0,0 +1,572 @@
...
+    
+   
+}
+
+main p{
+    letter-spacing: 0.1rem;
+}
+
+aside {
+
```
Comment (pnchinmay) [inline component/rating/rating.css:110]: remove these empty lines

File: component/rating/rating.css [hunk 5]
```diff
@@ -0,0 +1,572 @@
...
+aside #hamburger{
+grid-area: ham;
+display: none;
+
+}
+
+
+h3{
+    font-size: 1.5rem;
+    /* line-height: 2rem; */
```
Comment (pnchinmay) [inline component/rating/rating.css]: remove these comments

File: component/rating/rating.css [hunk 6]
```diff
@@ -0,0 +1,572 @@
...
+    grid-area: pitch;
+    font-size: 2rem;
+    margin: auto;
+    letter-spacing: 0.1rem;
+}
+
+
+hr{
+    border-top: 1px solid var(--primary-color); 
+  width: 50rem;
```
Comment (pnchinmay) [inline component/rating/rating.css:149]: indentation

File: component/rating/rating.css [hunk 7]
```diff
@@ -0,0 +1,572 @@
...
+footer{
+    grid-area: footer;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding: 0.5rem;
+    border-top: 1px solid var(--primary-color);
+    margin-top: 3rem;
+
```
Comment (pnchinmay) [inline component/rating/rating.css]: empty lines need to be reviewed

**Review by pnchinmay — APPROVED** [review]
Take care of the suggestions 
Everything else looks good :100:

**Review by sahiba0915 — APPROVED** [review]

**Pull request merged**